sys-libs/mesa/mesa-18.2.0rc2.recipe

```diff
@@ -0,0 +1,150 @@
+SUMMARY="Multi-platform GL implementation"
```

Comment:
Remove the duplicated recipe.

Reply:
Better to remove the other one, as you messed up the whitespace cleanup, or revert those commits.